src/RpcClient.ts

```diff
@@ -39,7 +40,7 @@ export abstract class RpcClient {
         if (!message.data
             || !message.data.status
             || !message.data.id
-            || (this._allowedOrigin !== '*' && message.origin !== this._allowedOrigin)) return;
+            || (this._allowedOrigin !== '*' && message.origin !== this._allowedOrigin)) return false;
```

Comment:
Return type annotation would be good.

Reply:
Typescript detects that automatically and also complains if an inherited function forgets it.